PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -46,6 +46,48 @@ static bool isAllDigits(const generic_string &str)
 	return std::all_of(str.begin(), str.end(), ::isdigit);
 }
 
+bool AutoCompletion::addLanguageCustomChars()
+{
+	if (_funcCalltip._additionalWordChar == TEXT(""))

```

Comment:
It is recommend to use [`empty()`](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#always-use-empty-for-testing-if-a-string-is-empty-or-not)